activestorage/lib/active_storage/engine.rb

```diff
@@ -25,7 +25,6 @@ class Engine < Rails::Engine # :nodoc:
 
     config.active_storage = ActiveSupport::OrderedOptions.new
     config.active_storage.previewers = [ ActiveStorage::Previewer::PopplerPDFPreviewer, ActiveStorage::Previewer::MuPDFPreviewer, ActiveStorage::Previewer::VideoPreviewer ]
-    config.active_storage.analyzers = [ ActiveStorage::Analyzer::VideoAnalyzer, ActiveStorage::Analyzer::AudioAnalyzer ]
```

Comment:
We should keep this as an empty array. Otherwise applications that used to do something like this would suddenly break

```ruby
config.active_storage.analyzers << CustomAnalyzer
```